float32 brake_pos
uint8 parking_brake_status
uint64 timestamp 
uint64 source_time

